### ISSUE TITLE
how to change the loss function of this model?

### ISSUE THREAD

**Issue by gjt9274** [eriklindernoren/PyTorch-YOLOv3#367]
there is a sample imbalance problem in my data(with  positive and negative ratio is 1:10)，so I wanna to change the loss function to **focal loss**.
anybody who knows where to change the code and how or another solution to solve my question? 
Thanks a lot

**Comment by songxiaoju** [comment]
I want to do this, too.

**Comment by gjt9274** [comment]
> I want to do this, too.

so, did you solve it?

**Comment by songxiaoju** [comment]
> > I want to do this, too.
> 
> so, did you solve it?

sorry,I can't deal with it.
